eng/testing/xunit/xunit.console.targets

```diff
@@ -87,11 +87,9 @@
     </ItemGroup>
 
     <!-- Workaround for https://github.com/xunit/xunit/issues/1651 -->
-    <ItemGroup Condition="'$(ArchiveTests)' != 'true'">
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.utility.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.reporters.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.utility.netcoreapp10.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.reporters.netcoreapp10.dll" />
+    <ItemGroup>
+      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net462\xunit.abstractions.dll" />
+      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net6.0\xunit.abstractions.dll" />
```

Comment:
This changes every other year and unfortunately msbuild doesn't emit a warning/error when an item remove operation fails. Please add a target that errrors when one of those items aren't found.

Reply:
Ok, this is about to go away anyhow since I see you worked to get the original issue fixed.  I can refine this workaround a bit until we can pick up a new release of xunit.  IMO the bigger issue to fix here is the `dotnet test` - I just noticed the double write problem was not fixed and corrected it while I was here - I'd like to merge this rather than wait for the new Xunit.